### ISSUE TITLE
Skeleton - WCAG rule 1.1.1, 1.3.1, & 1.3.2

### ISSUE THREAD

**Issue by KennyAtHPE** [grommet/grommet#7600]
Skeletons are purely visual and contain no relevant meaning for screen readers, so should be given aria-hidden=\"true\. [Supporting documentation](https://adrianroselli.com/2020/11/more-accessible-skeletons.html)

Because Skeletons are appearing on screen readers, it is breaking three WCAG rules:
- 1.1.1 Non-text content
- 1.3.1 Info and Relationships
     - Because skeletons are not hidden, they may mess with the structure of the page for screen readers.
- 1.3.2 Meaningful Sequence
     - Hide skeletons from screen readers to preserve logical flow.
